build.sh

```diff
@@ -0,0 +1,3 @@
+sbt clean compile stage package universal:packageZipTarball
+artefact=`ls -1 target/universal/*.tgz`
+echo "deployable artefact: $artefact"
```

Comment:
is this debug?

Reply:
when run, it will tell where the zipped deployable artefact is to copy to a remote machine

Reply:
+1